src/Transition/SupportFiles/Report Variables 9-3-0 to 9-4-0.csv

```diff
@@ -1,114 +1,546 @@
-Environmental Impact Fuel Oil #2 NH3 Emissions Mass,Environmental Impact FuelOilNo2 NH3 Emissions Mass,
-Environmental Impact Fuel Oil #2 NMVOC Emissions Mass,Environmental Impact FuelOilNo2 NMVOC Emissions Mass,
-Environmental Impact Fuel Oil #2 Hg Emissions Mass,Environmental Impact FuelOilNo2 Hg Emissions Mass,
-Environmental Impact Fuel Oil #2 Pb Emissions Mass,Environmental Impact FuelOilNo2 Pb Emissions Mass,
-Environmental Impact Fuel Oil #2 Water Consumption Volume,Environmental Impact FuelOilNo2 Water Consumption Volume,
-Environmental Impact Fuel Oil #2 Nuclear High Level Waste Mass,Environmental Impact FuelOilNo2 Nuclear High Level Waste Mass,
-Environmental Impact Fuel Oil #2 Nuclear Low Level Waste Volume,Environmental Impact FuelOilNo2 Nuclear Low Level Waste Volume,
-Environmental Impact Fuel Oil #1 Source Energy,Environmental Impact FuelOilNo1 Source Energy,
-Environmental Impact Fuel Oil #1 CO2 Emissions Mass,Environmental Impact FuelOilNo1 CO2 Emissions Mass,
-Environmental Impact Fuel Oil #1 CO Emissions Mass,Environmental Impact FuelOilNo1 CO Emissions Mass,
-Environmental Impact Fuel Oil #1 CH4 Emissions Mass,Environmental Impact FuelOilNo1 CH4 Emissions Mass,
-Environmental Impact Fuel Oil #1 NOx Emissions Mass,Environmental Impact FuelOilNo1 NOx Emissions Mass,
-Environmental Impact Fuel Oil #1 N2O Emissions Mass,Environmental Impact FuelOilNo1 N2O Emissions Mass,
-Environmental Impact Fuel Oil #1 SO2 Emissions Mass,Environmental Impact FuelOilNo1 SO2 Emissions Mass,
-Environmental Impact Fuel Oil #1 PM Emissions Mass,Environmental Impact FuelOilNo1 PM Emissions Mass,
-Environmental Impact Fuel Oil #1 PM10 Emissions Mass,Environmental Impact FuelOilNo1 PM10 Emissions Mass,
-Environmental Impact Fuel Oil #1 PM2.5 Emissions Mass,Environmental Impact FuelOilNo1 PM2.5 Emissions Mass,
-Environmental Impact Fuel Oil #1 NH3 Emissions Mass,Environmental Impact FuelOilNo1 NH3 Emissions Mass,
-Environmental Impact Fuel Oil #1 NMVOC Emissions Mass,Environmental Impact FuelOilNo1 NMVOC Emissions Mass,
-Environmental Impact Fuel Oil #1 Hg Emissions Mass,Environmental Impact FuelOilNo1 Hg Emissions Mass,
-Environmental Impact Fuel Oil #1 Pb Emissions Mass,Environmental Impact FuelOilNo1 Pb Emissions Mass,
-Environmental Impact Fuel Oil #1 Water Consumption Volume,Environmental Impact FuelOilNo1 Water Consumption Volume,
-Environmental Impact Fuel Oil #1 Nuclear High Level Waste Mass,Environmental Impact FuelOilNo1 Nuclear High Level Waste Mass,
-Environmental Impact Fuel Oil #1 Nuclear Low Level Waste Volume,Environmental Impact FuelOilNo1 Nuclear Low Level Waste Volume,
-old variable name,new variable name -- <DELETE> to delete,add variable names (before this line) and leave off units -- text in this column will add a warning
+9.3.0,9.4.0,Transition notes - some of these are EMS variable names,,,,,,
+543,543,These numbers should be the number of report variables in the following list (including deletes). Two columns/numbers.,,,,,,
```

Comment:
I don't know why this saved with all the extra columns.  It shouldn't be a problem, but I'll still try to clean this up later.